build/spas.js

```diff
@@ -263,13 +270,38 @@ async function buildSPAs(options) {
       if (!fs.statSync(path.join(root, locale)).isDirectory()) {
         continue;
       }
+
+      // circular dependency, so needs to be imported down here:
+      const { buildDocument } = require("./");
+      const featuredArticles = (
+        await Promise.all(
+          FEATURED_ARTICLES.map(async (url) => {
+            const document =
+              findByURL(`/${locale}/docs/${url}`) ||
+              findByURL(`/en-US/docs/${url}`);
+            if (document) {
+              const {
+                doc: { mdn_url, summary, title, parents },
+              } = await buildDocument(document);
+              return {
+                mdn_url,
+                summary: summary.split(/(\.(?=\s)|。)/i, 2).join(""), // cut at end of first sentence
+                title,
+                tag: parents[1],
```

Comment:
Just in case: provide a fallback.

Reply:
Did something a little different: made the component handle a null `tag`, and added a check here to avoid weird duplication like this:

![image](https://user-images.githubusercontent.com/755354/164510706-eb1a463d-08aa-4957-985e-e3b69b51deb2.png)

instead, if we ever wanted to feature a whole category of documentation, this would be rendered:

![image](https://user-images.githubusercontent.com/755354/164510917-1f9c87be-1eba-46f5-b406-f24d9388c953.png)

Thoughts?